core/templates/services/audio-bar-status.service.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service to notify about status of AudioBar
+ */
+
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AudioBarStatusService {
+  audioBarIsExpanded: boolean = false;
+
+  expandAudioBar(): void {
+    this.audioBarIsExpanded = true;
+  }
+
+  collapseAudioBar(): void {
+    this.audioBarIsExpanded = false;
+  }
+
+  isAudioBarExpanded(): boolean {
+    return this.audioBarIsExpanded;
+  }
+}
+
+angular.module('oppia').factory('AudioBarStatusService',
+  downgradeInjectable(AudioBarStatusService));
+
```

Comment:
is this an extra newline? Also, why this wasn't caught by linter?

@Hudda, can you please check whether this is expected?

Reply:
@DubeySandeep Yes, this is an extra newline. We do not have a check implemented for this yet. It is already added in #8423

Reply:
Done